strawberryfields/backends/base.py

```diff
@@ -475,38 +473,40 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
         Args:
             phi (float): phase angle of the quadrature to measure (x: :math:`\phi=0`, p: :math:`\phi=\pi/2`)
             mode (int): which mode to measure
-            select (float): (Optional) desired values of measurement results.
-                Allows user to post-select on specific measurement results instead of randomly sampling.
+            select (None or float): If not None: desired value of the measurement result.
+                Enables post-selection on specific measurement results instead of random sampling.
             **kwargs: can be used to pass user-specified numerical parameters to the backend.
                 Options for such arguments will be documented in the respective subclasses.
 
         Returns:
-            float: measured value
+            float or array[float]: measured value
         """
         raise NotImplementedError
 
     def is_vacuum(self, tol=0.0, **kwargs):
-        r"""Test whether the current circuit state is in vacuum (up to tolerance tol).
+        r"""Test whether the current circuit state is vacuum (up given tolerance).
+
+        FIXME The backends do not implement this method consistently, fockbackend uses np.linalg.norm whereas the other two use fidelity. Switch to fidelity for all?
 
         Args:
-            tol (float): numerical tolerance for how close state must be to true vacuum state
+            tol (float): numerical tolerance for how close state must be to true vacuum state (in terms of fidelity?)
```

Comment:
Let's fix this is up before merging this PR, so that it doesn't appear in the documentation

Reply:
Do you think fidelity would be a good measure here? @co9olguy

Reply:
yes, especially if 2/3 backends already use it

Reply:
Fidelity or distance in terms of the https://en.wikipedia.org/wiki/Bures_metric ?

Reply:
The difference seems to be a square root and a scaling of tol by 2

Reply:
Just default to whatever the current "majority vote" is amongst the backends that agree

Reply:
Fidelity it is then.